apps/client/src/components/main-page/SendChatMessage.tsx

```diff
@@ -6,13 +6,14 @@ import { selectUerData, setUserData } from '../../features/auth/authSlice';
 import AttachIcon from '../../assets/attach.svg';
 import SendIcon from '../../assets/send.svg';
 import { AttachFile } from './AttachFile';
-import { uploadData } from '../../utils';
+import { errorSwalMessage, uploadData } from '../../utils';
 
 export const SendChatMessage: FC<{ chatId: string }> = ({ chatId }) => {
   const inputRef = useRef<HTMLInputElement | any>();
   const [message, setMessage] = useState<string>('');
   const [isAttachFileShown, setIsAttachFileShown] = useState<boolean>(false);
-  const [readUrls, setReadUrls] = useState<string[]>([]);
+  const [files, setFiles] = useState<FileList | null>(null);
+  const [isAllowedToSend, setIsAllowedToSend] = useState<boolean>(false);
```

Comment:
unneeded state, you can derive this from message.length on the fly, maintaining less states

Reply:
Edited.